docs/cli/migration.md

```diff
@@ -0,0 +1,23 @@
+# Migration
```

Comment:
I was not entirely sure how to word a lot of this/what to include...

Reply:
We should highlight any command functionality that is in code42CLI that isn't yet or won't be here. 

Things like `devices activate/deactivate`, and showing backup data in `devices list`, `legal-hold` commands, etc.